OmniKit/PumpManager/OmnipodPumpManager.swift

```diff
@@ -287,18 +287,19 @@ extension OmnipodPumpManager {
         guard let podState = state.podState, let expiresAt = podState.expiresAt else {
             return nil
         }
+        let eolDisplayActiveTime = expiresAt.timeIntervalSinceNow + Pod.timeRemainingWarningThreshold
```

Comment:
`expiresAt` is the nominal expiration at 72 hours. `scheduledExpirationReminder` is the time of the user scheduled expiration reminder. I think you meant to use the latter?

Reply:
I believe OmniKit should match the OmniBLE version.  I'll hold off until the PR for OmniBLE is resolved. Then update this PR to be consistent.

Reply:
Furthermore this branch is out of date wrt dev and needs merge conflicts resolved.